Win11ThemeTest/TextBoxTest.cs

```diff
@@ -0,0 +1,416 @@
+                disabledTextBox = textWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt_disabled")).AsTextBox();
+                multiLineTextBox = textWindow.FindFirstDescendant(cf => cf.ByAutomationId("tbTxt_multiline")).AsTextBox();
+            }
+            catch (Exception ex)
+            {
+                var filePath = ConfigurationManager.AppSettings["logpath"];
+                if (filePath != null)
+                {
+                    if (!Directory.Exists(filePath))
+                    {
+                        Directory.CreateDirectory(filePath);
+                    }
+                    filePath = filePath + "log_" + DateTime.Now.ToString("yyyyMMddHHmmss") + ".txt";   //Text File Name
+                    if (!File.Exists(filePath))
+                    {
+                        File.Create(filePath).Dispose();
+                    }
+                    using (StreamWriter sw = File.AppendText(filePath))
+                    {
+                        string error = "Log Written Date:" + " " + DateTime.Now.ToString() + "\nError Message:" + " " + ex.Message.ToString();
+                        sw.WriteLine("-----------Exception Details on " + " " + DateTime.Now.ToString() + "-----------------");
+                        sw.WriteLine("-------------------------------------------------------------------------------------");
+                        sw.WriteLine(error);
+                        sw.Flush();
+                        sw.Close();
+                    }
+                }
+                else
+                {
+                    throw new ArgumentNullException();
+                }
+            }                           
+        }
+
+        [Test]
+        public void z_Cleanup()
+        {
+            Assert.IsNotNull(textWindow);
+            textWindow.Close();
+            Assert.IsTrue(textWindow.IsOffscreen);
+            Assert.IsNotNull(mainWindow);
+            mainWindow.Close();
+            Assert.IsTrue(mainWindow.IsOffscreen);
+        }
+
+        [Test]
+        public void tb_findTextBox()
+        {
+            Assert.IsNotNull(textWindow);           
+            Assert.IsNotNull(textBox);
+        }
+
+        #region FunctionalTests
+
+        [Test]
+        public void tb1_enterText()
+        {
+            Assert.IsNotNull(textBox);
+            textBox.Enter("Hello World!");
+           
+            Wait.UntilInputIsProcessed(TimeSpan.FromMilliseconds(500));
+            Assert.That(textBox.Text, Is.EqualTo("Hello World!"));
+        }
+        [Test]
+        public void multiLineTextbox()
+        {
+            string testDescription = "A good example of a paragraph contains a topic sentence, details and a conclusion. 'There are many different kinds of animals that live in China. Tigers and leopards are animals that live in China's forests in the north. Text.\nLine1\nLine2\nLine3";
+            multiLineTextBox.Enter(testDescription);         
```

Comment:
Address this
![image](https://github.com/singhashish-wpf/Win11ThemeSampleApp/assets/85492722/fbdf7ee0-fbc1-4210-be6d-8d12e0bb6f49)